relay-server/src/service.rs

```diff
@@ -26,6 +28,8 @@ use crate::middlewares::{
 };
 use crate::{endpoints, utils};
 
+pub static REGISTRY: OnceBox<Registry> = OnceBox::new();
```

Comment:
Please make this non-public and hide the implementation detail that this uses a `OnceBox`. Especially, this is considered infallible, and the `unwrap` should be hidden behind an accessor method:

```rust
impl Registry {
    fn get() -> &Self {
        REGISTRY.get().unwrap()
    }
}
```